packages/waku/src/lib/plugins/vite-plugin-rsc-rsdw.ts

```diff
@@ -21,7 +21,7 @@ globalThis.__WAKU_${type}_CHUNK_LOAD__ ||= (
   if (!globalThis.__WAKU_${type}_MODULE_LOADING__.has(id)) {
     globalThis.__WAKU_${type}_MODULE_LOADING__.set(
       id,
-      customImport(id).then((m) => {
+      (customImport ? customImport(id) : import(id)).then((m) => {
```

Comment:
I think this won't work because node condition is not including `react-server`

Reply:
that's why you need a vite server to load the module

Reply:
and import() doesn't support .ts(x) file in dev side

Reply:
Okay, but it should work on PRD, right?

Reply:
I think so, but ID is not correct; it is now like `/path/to/file` which is presented as HTTP resource